address-space-controller/src/main/resources/templates/standard-space-infra.yaml

```diff
@@ -207,10 +207,10 @@ objects:
       app: enmasse
       infraType: standard
       infraUuid: ${INFRA_UUID}
-      addressspace:
+      role: broker
```

Comment:
not sure if this is the right use of role.  Maybe role ought to be 'support-credentials'    @lulf do you have best practice advice to share?

Reply:
I agree that it looks a bit add having `role: broker` here. However, applying the same labels as those applied to corresponding broker pods allows the correct secret to be easily found by the system test. 
https://github.com/EnMasseProject/enmasse/pull/4276/files#diff-b037b734d02edac5aba40f8fd25d5d1eR88

Reply:
The content of CRs should be as meaningful and least surprise is possible. Changing the test to use a different set of labels is the right thing to do I think.

Reply:
Role is just a label we have defined to use for brokers, and not really a convention, just something that has always been that way. We don't even use the same label key for routers, so I'm fine just using 'role' here as well.